### PR TITLE
Theme search page in gent-base

### PR DESCRIPTION
**Pull request by hstandaert**
### Gent-base
- Style search form on `/search`
- styling fixes for opening hours
- Theme view mode `search_index`
- Removed help-text from `region--filters`, since the logic for displaying/hiding is integrated into the project

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [ ] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

File: source/sass/11-base/forms/_forms.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.views-exposed-form { // exposed view filters
+  .form-item {
+    margin-bottom: 0;
+    margin-right: .4rem;
+  }
+
+  // search form
+  .search {
```
Comment (meirege) [inline source/sass/11-base/forms/_forms.scss]: Is this generic enough for gent_base?

File: styleguide/components/31-molecules/frequently-visited/_frequently-visited.scss [hunk 2]
```diff
@@ -1,6 +1,12 @@
 .frequently-visited {
```
Comment (meirege) [inline styleguide/components/31-molecules/frequently-visited/_frequently-visited.scss:1]: Same here... Isn't this quite specific for Stad Gent?

File: source/sass/11-base/forms/_forms.scss (hunk 1 above)
Comment (hstandaert) [reply source/sass/11-base/forms/_forms.scss]: Don't think so; 90% of the projects have a search page, no? 🤔

File: styleguide/components/31-molecules/frequently-visited/_frequently-visited.scss (hunk 2 above)
Comment (hstandaert) [reply styleguide/components/31-molecules/frequently-visited/_frequently-visited.scss:1]: This is a styleguide component (added in #549)

File: styleguide/components/41-organisms/filter/filter.twig [hunk 3]
```diff
@@ -1,6 +1,8 @@
 <div class="sidebar-layout filter">
   {% set modalContent %}
     <h2>Filter the results</h2>
+    <p class="help-text">When selecting or deselecting filters, the page is automatically reloaded.</p>
```
Comment (delrueba) [inline styleguide/components/41-organisms/filter/filter.twig]: Best check with @mattiasberlamont if this is the default behaviour (styleguide) or a drupal-exception.
Either way "page reloaded" will only apply to drupal, others will only load the results with ajax (or wait for a submit).

File: source/sass/11-base/forms/_forms.scss (hunk 1 above)
Comment (delrueba) [reply source/sass/11-base/forms/_forms.scss]: They should, seeing as all pages must be reachable on "multiple ways".

But is base/forms the correct place?

File: styleguide/components/41-organisms/filter/filter.twig (hunk 3 above)
Comment (mattiasberlamont) [reply styleguide/components/41-organisms/filter/filter.twig]: @delrueba I think we spoke about this and concluded that the default behaviour (styleguide) should be like a form with one submit button, and thus without disclaimer. Correct? Or would you choose another default behaviour?

Comment (hstandaert) [reply styleguide/components/41-organisms/filter/filter.twig]: Agreed 🙂decision was made before my arrival here, so must've been a misunderstanding from my side. I'll remove it from styleguide and add it in gent-base

File: source/sass/11-base/forms/_forms.scss (hunk 1 above)
Comment (hstandaert) [reply source/sass/11-base/forms/_forms.scss]: I moved the styling for the search form specifically to a separate molecule, called search.

**Review by delrueba — APPROVED** [review]

**Review by meirege — APPROVED** [review]

**Pull request merged**